modules/weko-items-ui/weko_items_ui/utils.py

```diff
@@ -852,6 +852,50 @@ def get_list_file_by_record_id(recid):
     return list_file_name
 
 
+def write_bibtex_files(item_types_data, export_path):
+    """Write Bitex data to files.
+
+    @param item_types_data:
+    @param export_path:
+    @return:
+    """
+    for item_type_id in item_types_data:
+        item_type_data = item_types_data[item_type_id]
+        output = make_bibtex_data(item_type_data['recids'])
+        # create file to write data in case has output of Bibtex
+        if output:
+            with open('{}/{}.bib'.format(export_path,
+                                         item_type_data.get('name')),
+                      'w') as file:
+                file.write(output)
+
+
+def write_tsv_files(item_types_data, export_path, list_item_role):
```

Comment:
This method is a duplicated source code with the feature export JSON.
Please use this method for feature export JSON.